index.html

```diff
@@ -555,54 +573,86 @@ <h2>Sample Policy Definitions</h2>
+        the [[REPORTING]] API when uploading the report; the payload's
+        <code>body</code> field contains the <a>network error report body</a>.
+        </p>
 
         <pre class="example">
 {
-        "nel-report": [
-            {
-              "uri": "https://www.example.com/",
-              "referrer": "http://example.com/",
-              "server_ip": "123.122.121.120",
-              "protocol": "h2",
-              "status_code": 200,
-              "elapsed_time": 823,
-              "age": 0,
-              "type": "http.protocol.error"
-            }
-          ]
-        }
+  "age": 0,
+  "type": "network-error",
+  "url": "https://www.example.com/",
+  "body": {
+    "uri": "https://www.example.com/",
+    "sampling_fraction": 0.5,
+    "referrer": "http://example.com/",
+    "server_ip": "123.122.121.120",
+    "protocol": "h2",
+    "method": "GET",
+    "status_code": 200,
+    "elapsed_time": 823,
+    "type": "http.protocol.error"
+  }
+}
         </pre>
 
-        <p>The above report indicates that the user agent attempted to navigate from "example.com" to "www.example.com" (<a>known NEL origin</a>), which successfully resolved to the "123.122.121.120" IP address. However, while the user agent received a "200" response from the server via the "h2" protocol, it encountered a protocol error in the exchange and was forced to abandon the navigation. 823 milliseconds elapsed between the start of navigation and when the user agent aborted the navigation. Finally, the user agent sent this report immediately after the network error was encountered - i.e. the report age is 0.</p>
+        <p>
+        This report indicates that the user agent attempted to navigate from
+        <code>example.com</code> to <code>www.example.com</code> (<a>known NEL
+          origin</a>), which successfully resolved to the
+        <code>123.122.121.120</code> IP address.  However, while the user agent
+        received a <a>200 response</a> from the server via the HTTP/2
+        (<code>h2</code>) protocol, it encountered a protocol error in the
+        exchange and was forced to abandon the navigation.  The user agent
+        aborted the navigation 823 milliseconds after it started.  Finally, the
+        user agent sent this report immediately after the network error was
+        encountered – i.e. the report age is 0.
+        </p>
 
         <pre class="example">
 {
-        "nel-report": [
-            {
-              "uri": "https://widget.com/thing.js",
-              "referrer": "https://www.example.com/",
-              "server_ip": "234.233.232.231",
-              "protocol": "",
-              "status_code": 0,
-              "elapsed_time": 143,
-              "age": 0,
-              "type": "http.dns.name_not_resolved"
-            }
-          ]
-        }
+  "age": 0,
+  "type": "network-error",
+  "url": "https://widget.com/thing.js",
+  "report": {

```

Comment:
I believe this should be `body` instead of `report` based on https://wicg.github.io/reporting/#try-delivery ?

Reply:
Yep!  Good catch, done